### PR TITLE
Added pgbench to run from run_snafu --tool

### PR DESCRIPTION
**Pull request by amitsagtani97**
Depends-On: 288

### CODE REVIEW

File: pgbench_wrapper/trigger_pgbench.py [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+        print(output[0])
+        if len(documents) > 0:
+            self._summarize_data(data, self.run[0], self.uuid, self.database, self.pgb_vers)
+        print("\n")
+        print(documents)
+        print("\n")
+        if len(documents) > 0:
+            yield documents, 'summary'
+
+            if len(documents_raw) > 0:
```
Comment (aakarshg) [inline pgbench_wrapper/trigger_pgbench.py]: the documents_raw check should be independent of documents check imho... same for prog.. so maybe indent them out of the above loop?

File: pgbench_wrapper/pgbench_wrapper.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        self.args.cluster_name = "mycluster"
+        if "clustername" in os.environ:
+            self.args.cluster_name = os.environ["clustername"]
+        self.args.pgb_vers = subprocess.check_output("pgbench --version",
+                                                     shell=True).strip().decode("utf-8")
+        self.args.run_start_timestamp = datetime.now()
+        self.args.sample_start_timestamp = datetime.now()
+        self.args.index = "ripsaw-pgbench"
+
+        if "es" in os.environ:
```
Comment (aakarshg) [inline pgbench_wrapper/pgbench_wrapper.py]: these checks aren't needed...

**Comment by aakarshg** [comment]
ok to test

**Comment by aakarshg** [comment]
aborted the builds given that the index var is missing as pointed in ripsaw pr

File: pgbench_wrapper/trigger_pgbench.py [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+        documents = self._json_payload(self.meta_processed, data)
+        documents_raw = self._json_payload_raw(self.meta_processed, data)
+        documents_prog = self._json_payload_prog(self.meta_processed, progress, data)
+        print(output[0])
+        if len(documents) > 0:
+            self._summarize_data(data, self.run[0], self.uuid, self.database, self.pgb_vers)
+        print("\n")
+        print(documents)
+        print("\n")
+        if len(documents) > 0:
```
Comment (aakarshg) [inline pgbench_wrapper/trigger_pgbench.py]: this needs to be:
```
            for document in documents:
                yield document, 'summary'
```
We've to yield the individual document and not the list.

File: pgbench_wrapper/trigger_pgbench.py [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+            yield documents, 'summary'
+
+        if len(documents_raw) > 0:
```
Comment (aakarshg) [inline pgbench_wrapper/trigger_pgbench.py]: this needs to be:
```
            for document in documents_raw:
                yield document, 'summary'
```
We've to yield the individual document and not the list.

File: pgbench_wrapper/trigger_pgbench.py [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+            yield documents_raw, 'raw'
+
+        if len(documents_prog) > 0:
+            yield documents_prog, 'results'
```
Comment (aakarshg) [inline pgbench_wrapper/trigger_pgbench.py]: this needs to be:
```
            for document in documents_prog:
                yield document, 'summary'
```
We've to yield the individual document and not the list.

File: pgbench_wrapper/trigger_pgbench.py (hunk 4 above)
Comment (dry923) [reply pgbench_wrapper/trigger_pgbench.py]: @aakarshg this still accurate^

Comment (aakarshg) [reply pgbench_wrapper/trigger_pgbench.py]: yep L205 dealt with it so we should be good :)

Comment (dry923) [reply pgbench_wrapper/trigger_pgbench.py]: what about line 206? ```yield document, 'raw'``` vs ```yield document, 'summary'``` from your comment

Comment (aakarshg) [reply pgbench_wrapper/trigger_pgbench.py]: aaah I see, that was a bad copy paste example on my end, it should be `raw` indeed... L206 already addressess my comment, so this an be resolved.

Comment (dry923) [reply pgbench_wrapper/trigger_pgbench.py]: ack. thanks!

**Comment by aakarshg** [comment]
@amitsagtani97 can you please rebase this pr ?

**Comment by aakarshg** [comment]
this needs a rebase too

**Pull request merged**